### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by luiscontrerasdo**
logo for the asyncdata do dominicana entry in the support prometheus section for companies

### CODE REVIEW

File: content/support-training.html [hunk 1]
```diff
@@ -69,6 +69,13 @@ <h2>Commercial support</h2>
               <a href="https://acend.ch/en/trainings/prometheus/" class="btn btn-training btn-default btn-large" target="_blank">
                 <img src="../assets/commercial-support-logos/acend.svg" alt="acend">
               </a>
+
+            <div class="col-md-3 col-sm-6 doc-content tc-col">
```
Comment (juliusv) [inline content/support-training.html:73]: This section is in the wrong position, it is currently nested within the Acend logo section, it needs to go after the `</div>` of that one. Also, could you make sure that the indentation is just like for the other logo sections?

Comment (luiscontrerasdo) [reply content/support-training.html:74]: Completed

Comment (juliusv) [reply content/support-training.html:74]: Thanks, the opening div line is still indented differently than the closing though?

**Comment by luiscontrerasdo** [comment]
Hello,

Now I think they are aligned.

**Comment by juliusv** [comment]
:+1: Thank you, yes!

**Pull request merged**